src/streamlink/plugin/plugin.py

```diff
@@ -254,7 +255,7 @@ def streams(self, stream_types=None, sorting_excludes=None):
                                 key=partial(stream_type_priority,
                                             stream_types))
 
-        streams = {}
+        streams = OrderedDict()
```

Comment:
This seems to be unnecessary, no? Since you're creating another OrderedDict below, there's no reason to make this one an OrderedDict as well (and if memory serves the builtin dict is slightly more efficient).